src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -27,133 +29,300 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Represents Cdf Studio page Actions
+ * Represents Cdf Studio Page Actions
  */
 public class CdfStudioActions {
   private static final Logger logger = LoggerFactory.getLogger(CdfStudioActions.class);
-  public static CdfStudioLocators cdfStudioLocators;
 
   static {
-    cdfStudioLocators = SeleniumHelper.getPropertiesLocators(CdfStudioLocators.class);
+    SeleniumHelper.getPropertiesLocators(CdfStudioLocators.class);
   }
 
+  /**
+   * Select Data Pipeline type using the select dropdown
+   *
+   * @param option String: Batch or Realtime
+   */
   public static void selectDataPipelineType(String option) {
     ElementHelper.clickOnElement(CdfStudioLocators.dataPipelineTypeDropdown);
-    /* TODO: Replace JsExecutor click with normal click: https://cdap.atlassian.net/browse/CDAP-18885 */
-    ElementHelper.clickOnElementUsingJsExecutor(CdfStudioLocators.locateDataPipelineTypeDropdownOption(option));
+    CdfStudioLocators.locateDataPipelineTypeDropdownOption(option).click();
```

Comment:
why not using element helper here? Is it a hack to avoid notification alert? If yes, please add a comment

Reply:
Comment added.